### DISCUSSION TITLE
화이자랑 모더나 선택만은 없을까요?

### DISCUSSION THREAD

**Discussion by SHKNS** [changdoc/naver-vaccine-macro#202] [General]
아무거나하면 AZ나 얀센도 선택될꺼같은데

**Comment by changdoc** [comment]
백신 선택이 되어 있으면 해당 백신만 예약 신청 시도를 하게됩니다.

다만, 지도에서 매크로를 동작 시키고 계시다면 백신 종류 판단을 따로 하지 않고 예약 신청 페이지로 이동 시키고 있어, 예약 신청 페이지로 이동되어 매크로가 중단되는게 확인되면 수동으로 다시 돌리시면 되겠습니다.

**Comment by bicebersa** [comment]
20대라면 화면에 화이자랑 모더나만 뜰겁니다.
